net-misc/seafile-client/seafile-client-7.0.2.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils xdg-utils
```

Comment:
We usually leave one empty line below EAPI to follow skel format.

Reply:
Done.